server/tests/models/tags.spec.js

```diff
@@ -0,0 +1,67 @@
+import chai from "chai";
+import chaiAsPromise from "chai-as-promised";
+import models from "@models";
+import { tagData, articleData, userData, category } from "@fixtures";
+
+chai.use(chaiAsPromise);
+const { assert, expect } = chai;
+const { Tags, Categories, Users, Articles } = models;
+
+beforeEach(async () => {
+  await models.sequelize.sync({ force: true });
+});
+const tagDependencies = async () => {
+  const tagInstance = await Tags.create(tagData[0]);
+  const tagId = tagInstance.get("id");
+	const userInstance = await Users.create(userData[0]);
```

Comment:
What is going on with the indents here ?

Reply:
It is well aligned on my system, I guess Github is the one behaving funny.